Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/StepDefinitions/RelationshipTemplatesStepDefinitions.cs

```diff
@@ -39,6 +39,15 @@ public async Task GivenARelationshipTemplateCreatedByIdentity(string templateNam
             (await client.RelationshipTemplates.CreateTemplate(new CreateRelationshipTemplateRequest { Content = TestData.SOME_BYTES })).Result!;
     }
 
+    [Given($"a Relationship Template {RegexFor.SINGLE_THING} created by {RegexFor.SINGLE_THING} with {RegexFor.SINGLE_THING} max allocations")]
+    public async Task GivenARelationshipTemplateCreatedByIdentityWithMaxAllocations(string templateName, string identityName, string maxAllocations)
+    {
+        var client = _clientPool.FirstForIdentityName(identityName);
+        int? allocations = maxAllocations == "-" ? null : int.Parse(maxAllocations);
```

Comment:
I don't think we need the `-` case. If we don't want a maximum number of allocations, we would rather write something like "... without max allocations"